packages/shared/lib/core/network/constants/network.constant.ts

```diff
@@ -1,72 +1,60 @@
 import { BASE_TOKEN } from '../constants'
-import { NetworkProtocol, NetworkType } from '../enums'
+import { NetworkId, NetworkProtocol } from '../enums'
 import { INetwork } from '../interfaces'
 
-export const NETWORK: Readonly<{ [key in NetworkProtocol]?: { [key in NetworkType]?: INetwork } }> = {
-    [NetworkProtocol.IOTA]: {
-        [NetworkType.Mainnet]: {
-            id: 'iota-mainnet',
-            name: 'IOTA',
-            protocol: NetworkProtocol.IOTA,
-            type: NetworkType.Mainnet,
+export const NETWORK: Readonly<{ [key in NetworkId]?: INetwork }> = {
+    [NetworkId.Iota]: {
+        id: NetworkId.Iota,
+        name: 'IOTA',
+        protocol: {
+            version: 1,
+            networkName: 'iota',
```

Comment:
Maybe worth using the `NetworkProtocol` enum here and on l.29?

Reply:
Plan is to reduce use of that so I can eventually remove it